### ISSUE TITLE
Create and publish "Guidelines" section for the wedding cake website

### ISSUE THREAD

**Issue by jeffchew** [carbon-design-system/ibm-dotcom-library#210]
_chsanche created the following on Mar 26:_  

 # As a Digital Design System adopter, I need guidelines for me to follow, so that I can complete my tasks without confusion or contribute confidently because of I know how to do it.

### Notes

- We need to create the Guideline section on the website
- There are multiple guidelines we currently do not have and need to be crafted so we can publish the guidelines on the website. 

**(initial list)**

- [ ] Accessibility - Overview, Color, Keyboard, Developer.
- [ ] Content - General, Guidance, Glossary.
- [ ] Color - Overview, Usage.
- [ ] Iconography - Library, Usage Contribute.
- [ ] Layout
- [ ] Motion - Basics, Choreography, Resources.
- [ ] Spacing 
- [ ] Themes 
- [ ] Typography - Overview, Productive, Expressive.
- [ ] Contribution - Governance, Designers, Developers, Add-ons

### Acceptance criteria

- [ ] Our adaptor can find the guidelines they need on the DDS website.
- [ ] Guidelines will be published on the website under the "Guideline" section (coming soon).

### Tasks

- [ ] Review the guideline submission
- [ ] Review the guideline design
- [ ] Review the design with the developer
- [ ] Developer prep to write code 

_Original issue: https://github.ibm.com/webstandards/digital-design/issues/663_

**Comment by jeffchew** [comment]
_On May 21, chsanche commented:_
**A quick view on what Guidelines we currently have on the DDS website**

<img width="254" alt="screen shot 2019-05-21 at 12 44 29 pm" src="https://media.github.ibm.com/user/5181/files/55848500-7bc6-11e9-915a-ba47d3efb48d">
<img width="240" alt="screen shot 2019-05-21 at 12 44 59 pm" src="https://media.github.ibm.com/user/5181/files/561d1b80-7bc6-11e9-9b8d-8ab2ec6b1549">
<img width="239" alt="screen shot 2019-05-21 at 12 45 08 pm" src="https://media.github.ibm.com/user/5181/files/561d1b80-7bc6-11e9-9428-491a8bcf074a">

---

**A quick view on what Guidelines Carbon website currently has:**
<img width="250" alt="screen shot 2019-05-21 at 12 46 37 pm" src="https://media.github.ibm.com/user/5181/files/86fd5080-7bc6-11e9-9210-705c79458e0a">

**Comment by jeffchew** [comment]
_On May 21, hahnrob commented:_
@chsanche Could you be more specific in the Notes above the Acceptance Criteria?  A new issue has been linked to this epic to create an audit for the bands guidelines needed by MSC as soon as possible.

**Comment by jeffchew** [comment]
_On May 21, chsanche commented:_
@hahnrob This task issue is created to have the guidelines that are currently missing to be published on the website so our users can have the guidelines to guide them through whatever they need to do such as contribution guidelines for designers and developers so they know how to contribute back to DDS.

It is an Epic bc there are many guidelines we need to create and publish. The end goal is to have guidelines published on the website. I am not sure what detail is missing. Can you let me know so I can make it more specific as requested?

**Comment by jeffchew** [comment]
_On May 21, chsanche commented:_
The user story is not making sense - FYI. This task issue is created for DDS website guidelines creation but not just for bands and responsive behavior.

My recommendation is to not repurpose this Epic created for the website but create a new issue for the bands and responsive behavior.

**Comment by jeffchew** [comment]
_On Jun 24, chsanche commented:_
**Update:** This guideline section will be published on the new Gatsby website or use the one Carbon has based on the mental model/ website IA decision.

**Comment by jeffchew** [comment]
_On Jul 09, chsanche commented:_
**Note:** Contribution guidelines Carbon website currently has.
- [Governance](https://www.carbondesignsystem.com/contributing/governance)
- [Designers](https://www.carbondesignsystem.com/contributing/designers) 
- [Developers](https://www.carbondesignsystem.com/contributing/developers)
- [Add-ons](https://www.carbondesignsystem.com/contributing/add-ons)

---


![screen shot 2019-07-09 at 4 19 38 pm](https://media.github.ibm.com/user/5181/files/75026a80-a265-11e9-8162-012706a9a831)

**Comment by jeffchew** [comment]
_On Jul 22, Wonil-Suh1 commented:_
Adding the content strategy story as a dependency. Let's resume this epic once we have a content strategy.

**Comment by jeffchew** [comment]
_On Jul 22, chsanche commented:_
@Wonil-Suh1 Let's also keep in mind that Jan is working on the refinement of the guidelines for the Carbon website and before we start, let's make sure we aligned with what she is doing as well.
